packages/storage-postgres/src/schema/space.ts

```diff
@@ -15,6 +15,7 @@ export const spaces = pgTable('spaces', {
   categories: jsonb('categories')
     .$type<Array<(typeof categories.enumValues)[number]>>()
     .default([]),
+  parentId: serial('parent_id'),
   ...commonDateFields,
 });
 
```

Comment:
Please check the docs how to [declare relations](https://orm.drizzle.team/docs/relations#one-to-many)